### PR TITLE
Makes Collision 5 supports only Laravel 8

### PR DESCRIPTION
**Pull request by nunomaduro**
cc @GrahamCampbell

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
// cc @taylorotwell this will break the laravel/laravel develop branch?

**Comment by nunomaduro** [comment]
@GrahamCampbell I will test it.

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
         "filp/whoops": "^2.7.2",
         "symfony/console": "^5.0"
     },
+    "conflict": {
+        "laravel/framework": "<8.0"
```
Comment (GrahamCampbell) [inline composer.json:23]: Just below require dev asks for 7?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (nunomaduro) [reply composer.json:23]: That will change. Taking a better look at all those pull requests tomorrow or this weekend. Thanks!

**Comment by nunomaduro** [comment]
Done here: https://github.com/nunomaduro/collision/commit/ba26119149a7e42fbace9c09584596d313a535fd. Collision 5 artisan test command requires laravel 8.

**Pull request closed without merging**